benchmarks/perf-tool/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
```

Comment:
Could we add limits for each on the number of cpus and the max amount of memory the container can use? I believe it can be set via:
```
      resources:
        limits:
          cpus: '0.50'
          memory: 50M
        reservations:
          cpus: '0.25'
          memory: 20M
```
ref: https://docs.docker.com/compose/compose-file/compose-file-v3/#resources